### ISSUE TITLE
Mentor should be able to see tasks assigned to student

### ISSUE THREAD

**Issue by krzyk** [zerocracy/farm#1235]
Mentor should be able to see the agenda of student so he can inform about potential problems (like exceeding the delivery time).

I'm not sure if this should be available always, maybe until student achieves a defined number of points, e.g. 512? Or maybe the same limit as for allowing to join non-sandbox projects (128 ~256~)?

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@krzyk we can check that user from request is a mentor, but I'd hide `cost` field from mentor's view.

**Comment by krzyk** [comment]
@g4s8 yes, that makes sense

**Comment by krzyk** [comment]
@g4s8 Also I think it would be useful to see which projects where the student is involved. (e.g. to tell him to apply to given project). What do you think?
If it make sense, should I report it separately?

**Comment by g4s8** [comment]
@krzyk yes, I think it make. Please, edit title to make this ticket more generic, like "Mentor can't see student's profile" or something similar.

**Comment by krzyk** [comment]
@g4s8 done

**Comment by 0crat** [comment]
Job [#1235](https://github.com/zerocracy/farm/issues/1235) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)
